### PR TITLE
Update to work with the latest rust nightly (rustc 1.6.0-nightly (1727dee16 2015-11-26))

### PR DESCRIPTION
**Pull request by fhartwig**

### CODE REVIEW

File: regex_macros/src/lib.rs [hunk 1]
```diff
@@ -20,6 +20,7 @@
 extern crate regex;
 extern crate syntax;
 extern crate rustc;
```
Comment (hobofan) [inline regex_macros/src/lib.rs]: This line can actually be removed since rustc is only required for the plugin registry.

Comment (fhartwig) [inline regex_macros/src/lib.rs]: You're right, fixed. Thanks!

**Comment by untitaker** [comment]
Equivalent to #136

**Comment by BurntSushi** [comment]
Great, thank you!

**Pull request merged**